src/API.jl

```diff
@@ -1443,7 +1444,7 @@ end
 @deprecate status(mode::PackageMode) status(mode=mode)
 
 function status(ctx::Context, pkgs::Vector{PackageSpec}; diff::Bool=false, mode=PKGMODE_PROJECT,
-                io::IO=stdout, kwargs...)
```

Comment:
This should be `stdout` (this is output the user has asked for).

Reply:
or should it use `ctx.io` and remove the `io` kwarg?

Reply:
I get your point now.. I'll revert